### PR TITLE
Add basic Dockerfile

### PR DESCRIPTION
**Pull request by bn0ir**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+FROM node
+MAINTAINER bn0ir <gblacknoir@gmail.com>
+
+RUN apt-get update \
+    && apt-get install -y \
+       git \
+       sudo \
+       rsync \
```
Comment (msbrogli) [inline Dockerfile]: Why have you included the rsync dependency?

**Comment by msbrogli** [comment]
Thanks for your contribution. I'll merge it as soon as possible.

File: Dockerfile (hunk 1 above)
Comment (bn0ir) [inline Dockerfile]: Sorry, you right, there is no need for rsync in container.

**Comment by msbrogli** [comment]
Thanks! ;)

**Pull request merged**